examples/vic-gov-au/test/e2e/integration/core-modules/alert/alert.feature

```diff
@@ -13,31 +13,31 @@ Feature: Global Notifications (Alert)
       | 1-BE-Alert-1 Demo Alert | Demo Alert Type | https://vic.gov.au     | More  details |
   
   # Demo content module has an issue creating alert without primary site - skip test until that is fixed
-  @skip 
-  Scenario: 1-FE-Alert-2 Displays site section alert
-    Given I have created a node with the YAML fixture "/alert/1-BE-Alert-2"
-    When I visit the page "/"
-    Then there should be the following global notifications:
-      | title                   | type                | link                   | linkText      |
-      | 1-BE-Alert-1 Demo Alert | Demo Alert Type     | https://vic.gov.au     | More  details |
-    And I visit the page "/first-home-buyers-checklist"
-    And there should be the following global notifications:
-      | title                    | type               | link               | linkText           |
-      | 1-BE-Alert-1 Demo Alert  | Demo Alert Type    | https://vic.gov.au | More  details      |
-      | 1-BE-Alert-2 Demo Alert  | Heat Wave          | https://vic.gov.au | More  details      |
-    When I visit the page "/"
-    Then there should be the following global notifications:
-      | title                   | type            | link                | linkText      |
-      | 1-BE-Alert-1 Demo Alert | Demo Alert Type | https://vic.gov.au  | More  details |
+  # @skip 
```

Comment:
Commenting this out instead of deleting so it can be enabled once we work out how to create nodes without parent site taxonomy